tests/Unit/Utilities/Test_Math.cpp

```diff
@@ -43,4 +44,18 @@ SPECTRE_TEST_CASE("Unit.Utilities.Math", "[Unit][Utilities]") {
     CHECK(approx(invcbrt(27.0)) == (1 / 3.0));
     CHECK(approx(invcbrt(1.0 / 64.0)) == 4.0);
   }
+
+  SECTION("Test sign function") {
```

Comment:
We don't need the control flow of a 'SECTION' here. Better replace it with 'INFO' inside the scope.
Since this touches the whole test, we can make an issue instead.

Reply:
I think it was faster for me to make the change than to create an issue, so I opted for that :)